### PR TITLE
Fixed HTML reporter stats bug

### PR DESCRIPTION
**Pull request by kunagpal**
#581

### CODE REVIEW

File: lib/reporters/html/template-default.hbs [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <div class="row">
       {{#with summary}}
         <div class="col-md-3">Collection</div><div class="col-md-9">{{collection.name}}</div>
-        {{#if collection.description}}<div class="col-md-3">Description</div><div class="col-md-9">{{collection.description}}</div>{{/if}}
+        {{#if collection.description}}<div class="col-md-3">Description</div><div class="col-md-9">{{collection.description}}&nbsp;</div>{{/if}}
```
Comment (shamasis) [inline lib/reporters/html/template-default.hbs]: Any reason? If it lacks description why would that row occupy space?

**Comment by shamasis** [comment]
LGTM other than that one comment

File: lib/reporters/html/template-default.hbs (hunk 1 above)
Comment (kunagpal) [inline lib/reporters/html/template-default.hbs]: For empty collection descriptions, the row alignment is horribly skewed, such that the next line begins exactly where the description should have been.

Comment (shamasis) [inline lib/reporters/html/template-default.hbs]: KGTMW

**Pull request merged**